requirements.txt

```diff
@@ -0,0 +1,4 @@
+Flask==0.10
+icalendar==3.8
+mock==1.0
+python-dateutil==2.1
```

Comment:
Also I noticed you're using Flask 0.10.  Any reason for not using 0.10.1 (which we use in sheer)?